src/python/director/atlasdriverpanel.py

```diff
@@ -27,48 +29,84 @@ def __init__(self, driver):
         self.driver = driver
 
         loader = QtUiTools.QUiLoader()
+
+        # load Atlas by default
         uifile = QtCore.QFile(':/ui/ddAtlasDriverPanel.ui')
+
```

Comment:
the runstartup.py arrangement would allow us to avoid making this kind of logical fixes to switch between Atlas and Val